int32 MODE_INVALID_EXTERNAL_SETPOINT = -15
int32 MODE_FT_THRESHOLD_VIOLATION = -15
int32 MODE_SENSOR_INVALID_STATE=-14
int32 MODE_SENSOR_COMMUNICATION_ERROR = -13
int32 MODE_SENSOR_FAULT = -12
int32 MODE_COLLISION_IMMINENT = -11
int32 MODE_TRAJECTORY_ABORTED = -10
int32 MODE_TRAJECTORY_TRACKING_ERROR = -9
int32 MODE_INVALID_TRAJECTORY = -8
int32 MODE_SETPOINT_TRACKING_ERROR = -7
int32 MODE_ROBOT_INVALID_STATE = -6
int32 MODE_ROBOT_COMMUNICATION_ERROR = -5
int32 MODE_ROBOT_FAULT = -4
int32 MODE_INVALID_ARGUMENT = -3
int32 MODE_INVALID_OPERATION = -2
int32 MODE_INTERNAL_ERROR = -1
int32 MODE_HALT = 0
int32 MODE_JOINT_TELEOP = 1
int32 MODE_CARTESIAN_TELEOP = 2
int32 MODE_CYLINDRICAL_TELEOP = 3
int32 MODE_SPHERICAL_TELEOP = 4
int32 MODE_SHARED_TRAJECTORY = 5
int32 MODE_AUTO_TRAJECTORY = 6
int32 MODE_EXTERNAL_SETPOINT_0 = 7
int32 MODE_EXTERNAL_SETPOINT_1 = 8
int32 MODE_EXTERNAL_SETPOINT_2 = 9
int32 MODE_EXTERNAL_SETPOINT_3 = 10
int32 MODE_EXTERNAL_SETPOINT_4 = 11
int32 MODE_EXTERNAL_SETPOINT_5 = 12
int32 MODE_EXTERNAL_SETPOINT_6 = 13
int32 MODE_EXTERNAL_SETPOINT_7 = 14
int32 MODE_CUSTOM_MODE_0 = 65535
int32 MODE_SUCCESS = 2147483647

int32 mode
